app/Filament/Resources/NodeResource/Widgets/NodeMemoryChart.php

```diff
@@ -3,66 +3,78 @@
 namespace App\Filament\Resources\NodeResource\Widgets;
 
 use App\Models\Node;
+use Carbon\Carbon;
+use Filament\Support\RawJs;
 use Filament\Widgets\ChartWidget;
 use Illuminate\Database\Eloquent\Model;
 
 class NodeMemoryChart extends ChartWidget
 {
-    protected static ?string $heading = 'Memory';
-
-    protected static ?string $pollingInterval = '60s';
+    protected static ?string $pollingInterval = '5s';
+    protected static ?string $maxHeight = '300px';
 
     public ?Model $record = null;
 
-    protected static ?array $options = [
-        'scales' => [
-            'x' => [
-                'grid' => [
-                    'display' => false,
-                ],
-                'ticks' => [
-                    'display' => false,
-                ],
-            ],
-            'y' => [
-                'grid' => [
-                    'display' => false,
-                ],
-                'ticks' => [
-                    'display' => false,
-                ],
-            ],
-        ],
-    ];
-
     protected function getData(): array
     {
         /** @var Node $node */
         $node = $this->record;
 
-        $total = ($node->statistics()['memory_total'] ?? 0) / 1024 / 1024 / 1024;
-        $used = ($node->statistics()['memory_used'] ?? 0) / 1024 / 1024 / 1024;
-        $unused = $total - $used;
+        $memUsed = collect(cache()->get("nodes.$node->id.memory_used"))->slice(-10)
+            ->map(fn ($value, $key) => [
+                'memory' => $value / 1024 / 1024 / 1024,
```

Comment:
All values should be converted to/ from GB/ GiB based on the config value for the binary prefix. (`config('panel.use_binary_prefix')`)

Reply:
Why even give the option... Seems soo extra.

Reply:
Because most people want to use MB/ GB. They don't want MiB/ GiB.

Reply:
Isn't MegaByte, MB. 1024? That's what I'm using? I think our plan was to always use MB/GB

Reply:
MegaByte = MB = 1000 KB (decimal prefix)
MebiByte = MiB = 1024 KiB (binary prefix)

This is why there is a config option. It's very easy to mess these up.

Reply:
The latest commit should correctly use the config variable.